client/app/auth/auth.js

```diff
@@ -0,0 +1,39 @@
+angular.module('distress')
+
+//creates a factory called 'Auth' which we can
+//inject into our controllers
+.factory('Auth', function($http, $location){
+  var auth = {};
+
+  //sends a request to the /user/signin route of the server,
+  //still need to see what the server will respond with
+  auth.signin = function(user){
+    return $http({
+      method: 'GET',
```

Comment:
I think this should be a POST request as a user will be sending both a username and password, rather than a GET which is more for resource retrieval.